web/src/pages/project/ProjectGrid.tsx

```diff
@@ -60,29 +70,37 @@ const ProjectGrid: React.FunctionComponent<ProjectGridProps> = ({
         updateFilters(tempFilterValues)
     }
 
+    if (summary.participants.length === 0 && Object.keys(filterValues).length) {
+        headers = Object.entries(filterValues).map(([key, { category }]) => ({
+            name: key,
+            category,
+            title: category,
+        }))
+    }
+
     return (
         <Table celled>
             <SUITable.Header>
                 <SUITable.Row>
-                    {headers.map((k, i) => {
-                        if (k === 'Sample ID') {
+                    {headers.map(({ name, category, title }, i) => {
+                        if (title === 'Sample ID' || title === 'Created date') {
                             return (
                                 <SUITable.HeaderCell
-                                    key={`filter-${k}-${i}`}
+                                    key={`filter-${name}-${i}`}
                                     style={{ borderBottom: 'none' }}
                                 ></SUITable.HeaderCell>
                             )
                         }
                         return (
                             <SUITable.HeaderCell
-                                key={`filter-${k}-${i}`}
+                                key={`filter-${title}-${i}`}
                                 style={{ borderBottom: 'none' }}
                             >
                                 <div style={{ position: 'relative' }}>
                                     <div style={{ position: 'absolute', top: 0, right: 0 }}>
                                         <Popup
                                             trigger={
```

Comment:
Small request to make it easier to get into the search box
```suggestion
                                            position='top center'
                                            trigger={
```